pkg/cloudevents/types/value.go

```diff
@@ -131,6 +131,63 @@ func Validate(v interface{}) (interface{}, error) {
 	return nil, fmt.Errorf("invalid CloudEvents value: %#v", v)
 }
 
+// Validate v is a valid CloudEvents attribute value, convert it to one of:
+//     bool, int32, string, []byte, types.URI, types.URIRef, types.Timestamp
+func ValidateAndWrap(v interface{}) (interface{}, error) {
```

Comment:
See comment above - replace Validate() with this rather than adding a new function.